pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/entity/display/transform/TransformBuilder.kt

```diff
@@ -0,0 +1,123 @@
+package io.github.pylonmc.pylon.core.entity.display.transform
+
+import org.bukkit.Location
+import org.joml.*
+
+@Suppress("unused")
+class TransformBuilder(val components: ArrayDeque<TransformComponent>) {
+
+    constructor(): this(ArrayDeque())
+
+    constructor(other: TransformBuilder): this(ArrayDeque(other.components))
+
```

Comment:
```suggestion

    constructor(startingPos: Vector3f): this(ArrayDeque(setOf(Translation(startingPos))))
    
    constructor(startingPos: Vector3d): this(ArrayDeque(setOf(Translation(startingPos))))
    
```
just think this would be useful

Reply:
From my experience I've never needed this with display entities so will skip, think it's better to keep consistency